src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/ConnectHelper.cs

```diff
@@ -109,6 +109,9 @@ public static async ValueTask<QuicConnection> ConnectQuicAsync(HttpRequestMessag
             clientAuthenticationOptions = SetUpRemoteCertificateValidationCallback(clientAuthenticationOptions, request);
             QuicConnection connection = await QuicConnection.ConnectAsync(new QuicClientConnectionOptions()
             {
+                MaxBidirectionalStreams = 0, // Set to 100 if/when support for PUSH streams is added.
+                MaxUnidirectionalStreams = 5, // Should be enough for Control and QPack streams.
+                IdleTimeout = TimeSpan.FromMinutes(2), // To align with HttpClient default timeout.
```

Comment:
```suggestion
                IdleTimeout = HttpHandlerDefault.DefaultPooledConnectionIdleTimeout,
```
? (It's actually 1 minute)

Reply:
Fixed.